drivers/tty/serdev/pibridge_trace.h

```diff
@@ -248,12 +412,23 @@ DECLARE_EVENT_CLASS(pibridge_serdev_name_class,
 	)
 );
 
+/*
+ * pibridge_wakeup_write
+ *
+ * Info: the serdev device that is notified by the serdev subsystem that it is
+ *       ready to write more data.
+ * Time: at notification by the serdev subsystem.
+ */
 DEFINE_EVENT(pibridge_serdev_name_class, pibridge_wakeup_write,
 	TP_PROTO(struct serdev_device *serdev),
 	TP_ARGS(serdev)
 );
 
-/* packet data len */
+/*
+ * pibridge_data_len_class
+ *
+ * datalen: the number of bytes.
```

Comment:
s/datalen/len/

Reply:
Never mind, I didn't realize that you're specifying the data emitted by the tracepoint, not its arguments.